models/export.js

```diff
@@ -0,0 +1,95 @@
+/* global JsonRoutes */
+if(Meteor.isServer) {
+  // todo XXX once we have a real API in place, move that route there
```

Comment:
Another `XXX TODO` would be to share the route definition between the client and the server so that we could use something like `ApiRoutes.path('boards/export', boardId)` on the client instead of copy/pasting the route path manually between the client and the server.